ArduPlane/ArduPlane.cpp

```diff
@@ -732,25 +732,43 @@ bool Plane::set_target_location(const Location& target_loc)
 // set target location (for use by scripting)
 bool Plane::get_target_location(Location& target_loc)
 {
-    switch (control_mode->mode_number()) {
-    case Mode::Number::RTL:
-    case Mode::Number::AVOID_ADSB:
-    case Mode::Number::GUIDED:
-    case Mode::Number::AUTO:
-    case Mode::Number::LOITER:
+    target_loc = next_WP_loc;
+    return true;
```

Comment:
This is a change in behavior, that does not mean its wrong of course...

Reply:
after some thought I've changed it back